Makefile

```diff
@@ -38,7 +38,7 @@ binary-cross:
 
 all-yaml: kubeless.yaml kubeless-non-rbac.yaml kubeless-openshift.yaml kafka-zookeeper.yaml
 
-kubeless.yaml: kubeless.jsonnet
+kubeless.yaml: kubeless.jsonnet kubeless-non-rbac.jsonnet
```

Comment:
`kubeless-non-rbac.jsonnet` has its own task, why are you adding it here?

Reply:
Since `kubeless.yaml` is generated from `kubeless-non-rbac.jsonnet`, it should be a dependency when compiling.